include/networkit/io/NetworkitBinaryReader.hpp

```diff
@@ -24,12 +27,17 @@ class NetworkitBinaryReader final : public GraphReader {
     NetworkitBinaryReader(){};
 
     Graph read(const std::string &path) override;
+    Graph readState(const std::string &data);
```

Comment:
Rename to `readFromBuffer`. Let this take a `std::vector<uint8_t>` instead of a `std::string`.